### ISSUE TITLE
Exception when running tests in Github Actions

### ISSUE THREAD

**Issue by jbeuck** [spatie/laravel-ray#38]
Hey guys and thanks for the product!

I required Laravel Ray as an dev dependency and now my test's are failing when they are running in Github Actions. It seems like Ray is still trying to send logs to it's Client. Whichi think should'nt be the case when running in Github Actions.

`Exception: Ray seems not be running at http://localhost:23517`

Do i miss something in my configuration?

**Comment by freekmurze** [comment]
Could you post a back trace?

**Comment by jbeuck** [comment]
Sure
```
Tests\Unit\Imports\Gesellschaften\Ammerlaender\ImporterTest::it_tests_betrag_row with data set "Betrag string" (array('string'), null)
Exception: Ray seems not be running at http://localhost:23517

/home/runner/work/abrechnungstool/abrechnungstool/vendor/spatie/ray/src/Client.php:37
/home/runner/work/abrechnungstool/abrechnungstool/vendor/spatie/ray/src/Ray.php:327
/home/runner/work/abrechnungstool/abrechnungstool/vendor/spatie/laravel-ray/src/Ray.php:152
/home/runner/work/abrechnungstool/abrechnungstool/vendor/spatie/laravel-ray/src/RayServiceProvider.php:99
/home/runner/work/abrechnungstool/abrechnungstool/vendor/laravel/framework/src/Illuminate/Events/Dispatcher.php:391
/home/runner/work/abrechnungstool/abrechnungstool/vendor/laravel/framework/src/Illuminate/Events/Dispatcher.php:236
/home/runner/work/abrechnungstool/abrechnungstool/vendor/laravel/framework/src/Illuminate/Log/Logger.php:210
/home/runner/work/abrechnungstool/abrechnungstool/vendor/laravel/framework/src/Illuminate/Log/Logger.php:176
/home/runner/work/abrechnungstool/abrechnungstool/vendor/laravel/framework/src/Illuminate/Log/Logger.php:87
/home/runner/work/abrechnungstool/abrechnungstool/app/Imports/AbstractImporter.php:137
/home/runner/work/abrechnungstool/abrechnungstool/tests/Unit/Imports/Gesellschaften/Ammerlaender/ImporterTest.php:79
```

`AbstractImporter.php:137` here is where we Log something and it seems like ray is trying to send this log to it's client

**Comment by chriischambers** [comment]
I can confirm this happens in BitBucket pipelines too.

**Comment by freekmurze** [comment]
This has been fixed in spatie/ray 1.3.1 👍

**Comment by jbeuck** [comment]
thanks for your quick fix @freekmurze :heart:
